frontend/src/scenes/organizationLogic.tsx

```diff
@@ -20,18 +26,46 @@ export const organizationLogic = kea<organizationLogicType<OrganizationType, Per
                         return null
                     }
                 },
-                createOrganization: async (name: string) => await api.create('api/organizations/', { name }),
+                createOrganization: async (name: string) => {
+                    const result = await api.create('api/organizations/', { name })
+                    teamLogic.actions.loadCurrentTeam()
```

Comment:
You're re-loading these endpoints, but there's still a listener that will cause a hard refresh, rendering this unused.

Reply:
I'll keep this as we'll be moving towards not requiring a hard refresh (#3421).